i18n/versioned_docs/ja-jp/docusaurus-plugin-content-docs/version-3.8.json

```
@@ -1,6 +1,6 @@
 {
   "version.label": {
-    "message": "3.8",
+    "message": "3.8 (サポートされていない)",
```

```suggestion
    "message": "3.8 (サポート外)",
```
It's not directly related to this PR, so it can be done in another PR.

---

Thanks! Since it's an easy change specific to this repository, I've updated the wording XXX.